### PR TITLE
add weak external patches: https://sourceware.org/ml/binutils/2015-10…

### PR DESCRIPTION
**Pull request by staalmannen**
…/msg00234.html

### CODE REVIEW

**Comment by Alexpux** [comment]
Please increase pkgrel field too

**Comment by Alexpux** [comment]
Are your patches for binutils 2.26 or 2.25?

**Comment by Alexpux** [comment]
They fail to apply

**Comment by staalmannen** [comment]
hi sorry they are indeed for 2.26. I built a git variant locally and forgot
to change source in the package.

away today but will update tomorrow
Den 3 mars 2016 06:59 skrev "Алексей" <notifications@github.com>:

**Comment by staalmannen** [comment]
Hi updated the PKGBUILD to 2.26. msys2 target patch no longer needed, but I just commented it out

**Comment by staalmannen** [comment]
Sorry about all commits - my brain was not working today :) Now it passes the tests so it is not only building on my machine :)

File: binutils/PKGBUILD [hunk 1]
```diff
@@ -17,18 +17,30 @@ options=('staticlibs' '!distcc' '!ccache')
...
-            '4661ea9608e0c00c95dff5a43e4db69917a16b2877f4fae2e4f4330a5ec592be'
-            'bd38317b28b894d5c0e0ba043fc2445af415f6e96bb9ac93c348748d21a43625')
+            'bd38317b28b894d5c0e0ba043fc2445af415f6e96bb9ac93c348748d21a43625'
+            '76663a2493f81e2306140259a2e4306b7cc4592fc4381951df87b5aa52c0fb62'
+            'a5764911df3bde0e51fe2c244cac38a2f4169081017796b8cc9f214d34c17930'
+            '05933fb77e5577257fe1d7a6ff2e701e93fd43ce417a29165bc6a71da5326459')
 
 prepare() {
   cd "${srcdir}"/binutils-$_basever
-  patch -p1 -i "${srcdir}"/binutils-trunk-msys2.patch
```
Comment (Alexpux) [inline binutils/PKGBUILD]: Back msys2 patch.

File: binutils/PKGBUILD [hunk 2]
```diff
@@ -17,18 +17,30 @@ options=('staticlibs' '!distcc' '!ccache')
...
+  patch -p1 -i "${srcdir}"/coff-linker-weak-nt-externals.patch
+  patch -p1 -i "${srcdir}"/gas-aux-nt-weak-externals.patch
+  patch -p1 -i "${srcdir}"/objcopy-weak-nt-externals2local.patch
   patch -p1 -i "${srcdir}"/0005-bfd-Increase-_bfd_coff_max_nscns-to-65279.patch
 
+  #replacement for msys2 patch
+  sed -i 's|cygwin|msys|g' bfd/config.bfd
+  sed -i 's|cygwin|msys|g' gas/configur*
+  sed -i 's|cygwin|msys|g' ld/configur*
+
```
Comment (Alexpux) [inline binutils/PKGBUILD]: Remove this

**Comment by staalmannen** [comment]
Thanks for your feedback!
Fixed msys2 patch + squashed commits

**Pull request merged**